Source/ACE.Server/WorldObjects/Managers/EmoteManager.cs

```diff
@@ -116,16 +116,21 @@ public float ExecuteEmote(BiotaPropertiesEmote emoteSet, BiotaPropertiesEmoteAct
 
                 case EmoteType.AwardLevelProportionalSkillXP:
 
+                    var max = emote.Max64 ?? emote.Max ?? 0;
```

Comment:
can you put min before max for all of these? (definition, param order)

Reply:
k